distribution/src/conf/rule-engine-config.xml

```diff
@@ -0,0 +1,12 @@
+<ruleEngineConfig xmlns="http://wso2.org/carbon/rules/conf">
+    <!--<ruleEngineProvider className="org.wso2.carbon.rule.backend.jsr94.JSR94BackendRuntimeFactory">
+        <property name="propertyProvider"
+                  value="org.wso2.carbon.rule.backend.jsr94.drools.DroolsJSR94DefaultPropertiesProvider"/>
```

Comment:
@tharinduwijewardane  Is there any reason to specifically adding this file here rather than suing the one coming from the kernel.
https://github.com/wso2/product-ei/blob/master/distribution/src/assembly/bin.xml#L2333.